2022-01 January.md

```diff
@@ -0,0 +1,17 @@
+# January 2022
+
+I set out to substitute configd with the SystemConfiguration framework from the
```

Comment:
It would be good to mention here that all of the work was spent on SDK related activities.

Reply:
Added a line mentioning it.